### PR TITLE
17774 remove simplesamlphp salt and admin password from vagov cms and changes values

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Relates to #17774

## Testing done
Tested on test infrastructure

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

Cannot be QAd on tugboat. If it passes all automated testing it should be good. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [x] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: simplesamlphp-config-metadata/config/config.php [hunk 1]
```diff
@@ -188,26 +189,26 @@
      * A possible way to generate a random salt is by running the following command from a unix shell:
      * LC_ALL=C tr -c -d '0123456789abcdefghijklmnopqrstuvwxyz' </dev/urandom | dd bs=32 count=1 2>/dev/null;echo
      */
-    'secretsalt' => 'p97p6wmkz8flefjij0v4q1z643gxpa5j',
+  'secretsalt' => getenv('SIMPLESAMLPHP_SECRETSALT'),
```
Comment (timcosgrove) [inline simplesamlphp-config-metadata/config/config.php:192]:
```suggestion
    /*
     * We pull the SimpleSAMLphp's salt and admin password from environment variables, 
     * which in turn are pulled from AWS Parameter Store during deployment.
     * This prevents exposing the salt and password to the public.
     */ 
  'secretsalt' => getenv('SIMPLESAMLPHP_SECRETSALT'),
```

Comment (timcosgrove) [reply simplesamlphp-config-metadata/config/config.php:192]: Or incorporate this into the main comment above the salt assignment.

File: simplesamlphp-config-metadata/config/config.php [hunk 2]
```diff
@@ -188,26 +189,26 @@
...
 
     /*
      * This password must be kept secret, and modified from the default value 123.
      * This password will give access to the installation page of SimpleSAMLphp with
      * metadata listing and diagnostics pages.
      * You can also put a hash here; run "bin/pwgen.php" to generate one.
      */
-    'auth.adminpassword' => 'drupal8',
+  'auth.adminpassword' => getenv('SIMPLESAMLPHP_ADMIN_PASSWORD'),
```
Comment (timcosgrove) [inline simplesamlphp-config-metadata/config/config.php:200]:
```suggestion
    /*
     * Pulled from environment variables for secrecy. See the comment on secretsalt above.
     */
  'auth.adminpassword' => getenv('SIMPLESAMLPHP_ADMIN_PASSWORD'),
```

**Review by timcosgrove — COMMENTED** [review]
Added suggestions for comments to explain what's happening.

File: simplesamlphp-config-metadata/config/config.php (hunk 1 above)
Comment (edmund-dunn) [reply simplesamlphp-config-metadata/config/config.php:196]: added

File: simplesamlphp-config-metadata/config/config.php (hunk 2 above)
Comment (edmund-dunn) [reply simplesamlphp-config-metadata/config/config.php:206]: added

**Review by timcosgrove — APPROVED** [review]

**Pull request merged**